### ISSUE TITLE
Ansible 2.8.0 support

### ISSUE THREAD

**Issue by koaps** [oracle/oci-ansible-modules#53]
## Issue Report

When trying to install the module I'm getting the following:

```
bash-4.2# pwd
/opt/oci-ansible-modules

bash-4.2# ./install.py
Documentation fragments directory /usr/lib/python2.7/site-packages/ansible/utils/module_docs_fragments does not exist
```

Able to continue by creating the directory
```
bash-4.2# mkdir /usr/lib/python2.7/site-packages/ansible/utils/module_docs_fragments
```

**Expected behavior**

```
bash-4.2# ./install.py
Copying documentation fragments from /opt/oci-ansible-modules/module_docs_fragments to /usr/lib/python2.7/site-packages/ansible/utils/module_docs_fragments
Copying oracle utility files from /opt/oci-ansible-modules/module_utils/oracle to /usr/lib/python2.7/site-packages/ansible/module_utils/oracle
Overwriting /usr/lib/python2.7/site-packages/ansible/module_utils/oracle/oci_utils.py
Overwriting /usr/lib/python2.7/site-packages/ansible/module_utils/oracle/__init__.py
Copying OCI Ansible modules from /opt/oci-ansible-modules/library to /usr/lib/python2.7/site-packages/ansible/modules/cloud/oracle
Overwriting /usr/lib/python2.7/site-packages/ansible/modules/cloud/oracle/oci_vcn.py
Overwriting /usr/lib/python2.7/site-packages/ansible/modules/cloud/oracle/__init__.py
OCI Ansible modules installed successfully.
```

**Environment**

Docker container image: `oraclelinux:7-slim`

* OS version:

```
bash-4.2# cat /etc/redhat-release
Red Hat Enterprise Linux Server release 7.6 (Maipo)

bash-4.2# cat /etc/oracle-release
Oracle Linux Server release 7.6
```

* Ansible version:

```
bash-4.2# ansible --version
ansible 2.8.0
  config file = None
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Apr  9 2019, 16:02:27) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36.0.1)]
```

* OCI Python SDK version:

```
bash-4.2# python -c "import oci;print(oci.__version__)"
2.1.7
```

* OCI Ansible Modules version:

```
bash-4.2# python -c "from ansible.module_utils.oracle import oci_utils; print(oci_utils.__version__)"
1.6.0-dev
```

**Ansible playbook to reproduce the issue**

N/A - this is installation

**Comment by manojmeda** [comment]
Thanks @koaps for reporting this. As a workaround can you please use ansible galaxy to install our modules till this issue is fixed. Please let us know if you need any help regarding that.

**Comment by nalsaber** [comment]
@koaps this issue will be fixed soon in the upcoming release. Ansible has changed the path for the documentation files that's why the installation script is failing.

**Comment by koaps** [comment]
thanks @nalsaber 

@manojmeda thanks for the workaround, unfortunately our current setup doesn't use galaxy and introducing that now might cause problems, since we are building a docker image I can just add a step to create the directory as a workaround for now.

**Comment by nalsaber** [comment]
This issue was resolved in release  [v1.9.0](https://github.com/oracle/oci-ansible-modules/releases/tag/v1.9.0)
